src/ng/location.js

```diff
@@ -749,7 +749,9 @@ function $LocationProvider() {
    *     true, and a base tag is not present, an error will be thrown when `$location` is injected.
    *     See the {@link guide/$location $location guide for more information}
    *   - **rewriteLinks** - `{boolean}` - (default: `true`) When html5Mode is enabled,
-   *     enables/disables url rewriting for relative links.
+   *     enables/disables url rewriting for relative links. Note that if this is disabled, you can
+   *     still force a link to be rewritten by adding the "ng-noserver" attr to the `<a>` element:
+   *     `<a href="somewhere" ng-noserver>click me</a>`.
```

Comment:
Another idea would be allowing the users to set `rewriteLinks` to a string, indicating the attribute that forces rewriting. E.g.:

- `rewriteLinks: true`: Always rewrite links.
- `rewriteLinks: false`: Never rewrite links.
- `rewriteLinks: 'some-attr'`: Only rewrite links that have the `some-attr` attribute.